src/Interpreters/JIT/CHJIT.cpp

```diff
@@ -485,13 +493,30 @@ std::string CHJIT::getMangledName(const std::string & name_to_mangle) const
 
 void CHJIT::runOptimizationPassesOnModule(llvm::Module & module) const
 {
+    const char *argv[] =
```

Comment:
1. Calling LLVM's command line parser somewhere during query processing looks like a hack.
2. Can the extra vectorizer pass somehow be enabled somewhere down at l. 512ff?

Reply:
For 1, yes, so I directly change the value in llvm source code, see https://github.com/ClickHouse/llvm-project/pull/49.
For2, I read the code, there is no way to do it, see the code https://github.com/ClickHouse/llvm-project/blob/cc38ec02c99791f6591b378481bfb879c1b4eb0b/llvm/lib/Passes/PassBuilderPipelines.cpp#L192.